docs/how-to/framework.rst

```diff
@@ -193,7 +193,6 @@ Follow these steps:
 
 4. To see and run a sample training script, refer to `rocAL TensorFlow example <https://github.com/ROCm/rocAL/tree/master/rocAL/docs/examples/tf/pets_training>`_.
 
-
 .. __resnet50:
 
 Run Resnet50 classification training with rocAL
```

Comment:
Sorry, found another broken link.  Same problem, extra rocAL in path:
"rocAL Imagenet example", 20 lines below this one.  Should link to: https://github.com/ROCm/rocAL/tree/master/docs/examples/pytorch/imagenet_training